tests/integration/shell/auth.py

```diff
@@ -7,18 +7,37 @@
 # Import python libs
 import os
 import pwd
+import grp
 import random
 
 # Import Salt Testing libs
+from salttesting import skipIf
 from salttesting.helpers import (
     ensure_in_syspath,
     destructiveTest)
 ensure_in_syspath('../../')
 
 # Import salt libs
+from salt.utils.pycrypto import gen_hash
 import integration
 
-from salttesting import skipIf
+
+def gen_password():
+    '''
```

Comment:
@isbm, I know there is a reason for this, although I don't know what it is.  The salt style is to use single quotes, although I don't think our linter checks for that.